src/core/services/gesture/gesture.js

```diff
@@ -243,6 +244,18 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
         horizontal: true,
         cancelMultiplier: 1.5
       },
+      onSetup: function(element, opts) {
+        if (touchActionProperty) {
+          // We check for horizontal to be false, because otherwise we would overwrite the default opts.
+          this.oldTouchAction = element[0].style[touchActionProperty];
+          element[0].style[touchActionProperty] = opts.horizontal === false ? 'pan-y' : 'pan-x';
```

Comment:
The problem is, that `jqLite` is transforming the name from snake-case into camelCase, which breaks some browsers prefixes.

https://github.com/angular/angular.js/blob/master/src/jqLite.js#L145
![image](https://cloud.githubusercontent.com/assets/4987015/14226425/007560f8-f8e2-11e5-9f0f-2bd18288b640.png)